packages/react-native-reanimated/src/ReanimatedModule/index.web.ts

```diff
@@ -0,0 +1,4 @@
+'use strict';
+// this file was created to prevent NativeReanimated from being included in the web bundle
+import { createJSReanimatedModule } from './js-reanimated';
+export const ReanimatedModule = createJSReanimatedModule();
```

Comment:
Previously, we used `export ... from ...` which works correctly with tree-shaking. Can you confirm that the new code also supports tree-shaking?

Reply:
I'll check it.

Reply:
It works! 👍